connect.yaml

```diff
@@ -3,27 +3,28 @@ deployAs:
     applicationType: service
     endpoint: /
     configuration:
-      securedConfiguration:
+      standardConfiguration:
         - key: CTP_PROJECT_KEY
           description: Commercetools project key
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
-        - key: CTP_CLIENT_SECRET
-          description: Commercetools client secret
         - key: CTP_SCOPE
           description: Commercetools client scope
-        - key: CTP_REGION
+         - key: CTP_REGION
           description: Region of Commercetools project
```

Comment:
```suggestion
```
Should we remove the region? Not sure where it is used

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/67/commits/8644b9c96b81b2d84055a2f61edf994a8c7ccd2d